tests/forward_index_test.cpp

```diff
@@ -183,6 +183,21 @@ go_bandit([]() {
             line_cfg->insert("uninvert", true);
             ceeaus_forward_test(*line_cfg);
         });
+
+        it("should analyze a new document with the current analyzer", [&]() {
+            auto cfg = tests::create_config("line");
+            auto idx = index::make_index<index::forward_index>(*cfg);
+            std::string text{"I think smoking smoking bad."};
+            corpus::document doc;
+            doc.content(text);
+            auto fvector = idx->tokenize(doc);
+            AssertThat(fvector.at(term_id{0}), Equals(1));
+            AssertThat(fvector.at(term_id{1}), Equals(1));
+            AssertThat(fvector.at(term_id{276}), Equals(1));
+            AssertThat(fvector.at(term_id{3343}), Equals(2));
+            AssertThat(fvector.at(term_id{3731}), Equals(1));
+            AssertThat(fvector.at(term_id{2}), Equals(0));
```

Comment:
This test seems very fragile. Could we look up the term id for the specific terms via `disk_index::get_term_id()` instead of using hard-coded term ids?